core/packages/PackageInfo.cc

```diff
@@ -247,9 +247,9 @@ optional<core::AutocorrectSuggestion> PackageInfo::addImport(const core::GlobalS
         // exports, then right before the final `end`
         int64_t exportLoc;
         if (!exports_.empty()) {
-            exportLoc = exports_.front().loc.beginPos() - "export "sv.size() - 1;
+            exportLoc = exports_.front().loc.beginPos() - "\n"sv.size();
```

Comment:
This looks like it's right in front of the export, so perhaps `" "` is more appropriate? The value will be the same, but it's nice documentation.
```suggestion
            exportLoc = exports_.front().loc.beginPos() - " "sv.size();
```

Reply:
I wonder if that's a bug then